### PR TITLE
(Article -> Security) Official domains

### PR DESCRIPTION
**Pull request by 409H**
* Added document detailing official domains and communications

### CODE REVIEW

File: security/official-domains.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+This domain is for staging and QA to test builds for our blockchain interface. Since the code 
+running the interface is in test phase, there may be bugs - don't use it with the same
+conviction as production (`mycrypto.com`).
+
+### Verified mycryptobuilds.com domains
+
+* `mycryptobuilds.com` - serves QA builds for the blockchain interface
+
+### Verified mycryptoapi.com domains
```
Comment (Mrtenz) [inline security/official-domains.md]: This section is in there twice.

File: security/official-domains.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+* `proxy.mycryptoapi.com` - handles proxying requests to third-parties in the background
+* `raidentransport.mycryptoapi.com` - [matrix] server for Raiden related stuff
+* `api.mycryptoapi.com` - handles requests to our Blockchain RPC nodes
+* `apis.mycryptoapi.com` - used for weighted load balancing between two regions
+
+
```
Comment (Mrtenz) [inline security/official-domains.md]: This line should be removed for Travis to pass

File: security/official-domains.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+## Official email from MyCrypto
+
+If you recieve a communication that looks like it's from MyCrypto but _doesn't_ come from
```
Comment (Mrtenz) [inline security/official-domains.md]: `_doesn't_` -> `*doesn't*`

File: security/official-domains.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+### Verified mycryptoapi.com domains
+
+* `proxy.mycryptoapi.com` - handles proxying requests to third-parties in the background
+* `raidentransport.mycryptoapi.com` - [matrix] server for Raiden related stuff
+* `api.mycryptoapi.com` - handles requests to our Blockchain RPC nodes
+* `apis.mycryptoapi.com` - used for weighted load balancing between two regions
+
+
+## Official email from MyCrypto
```
Comment (Mrtenz) [inline security/official-domains.md]: Maybe add a line about the newsletter?

File: security/official-domains.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
+---
+title: Official Domains
```
[the remark this answers is not in the document]
Comment (409H) [reply security/official-domains.md]: Updated in https://github.com/MyCryptoHQ/knowledge-base-content/pull/33/commits/c2d81736ceb6c173e191cc793222deda4e72b4c0

File: security/official-domains.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+## mycrypto.com
+
+Most of the things a typical user is interested on will be on the `mycrypto.com` domain - including various subdomains.
+
+### Verified mycrypto.com domains
+
+All MyCrypto content should originate from one of the following domains;
+
+* `mycrypto.com` - the production-stable blockchain interface
+* `about.mycrytpo.com` - team profile and job opportunities
```
[the remark this answers is not in the document]
Comment (409H) [reply security/official-domains.md]: Updated in https://github.com/MyCryptoHQ/knowledge-base-content/pull/33/commits/c2d81736ceb6c173e191cc793222deda4e72b4c0

File: security/official-domains.md (hunk 1 above)
Comment (409H) [reply security/official-domains.md]: Removed in https://github.com/MyCryptoHQ/knowledge-base-content/pull/33/commits/c2d81736ceb6c173e191cc793222deda4e72b4c0

File: security/official-domains.md (hunk 3 above)
Comment (409H) [reply security/official-domains.md]: Updated in https://github.com/MyCryptoHQ/knowledge-base-content/pull/33/commits/c2d81736ceb6c173e191cc793222deda4e72b4c0

File: security/official-domains.md (hunk 4 above)
Comment (409H) [reply security/official-domains.md]: Updated in https://github.com/MyCryptoHQ/knowledge-base-content/pull/33/commits/c2d81736ceb6c173e191cc793222deda4e72b4c0

**Pull request merged**